_1327/polls/models.py

```diff
@@ -53,7 +53,13 @@ def generate_new_title(cls):
 
 	@classmethod
 	def generate_default_slug(cls, title):
-		return "polls_{}".format(slugify(title))
+		slug_start = "polls_{}".format(slugify(title))
```

Comment:
please use `new-poll-from-` and add the date instead of `polls_` and adding the title